packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -640,6 +666,14 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     ISummarizerInternalsProvider {
     public get IContainerRuntime() { return this; }
     public get IFluidRouter() { return this; }
+    private static buildSummaryConfiguration(runtimeOptions: IContainerRuntimeOptions): ISummaryConfiguration {
```

Comment:
Nit: Recommend inlining `buildSummaryConfiguration` now that we're back to a single caller.  (Improves local reasoning.)